Jellyfin.Plugin.AniDB/Providers/equals_check.cs

```diff
@@ -21,527 +19,212 @@ public Equals_check(ILogger<Equals_check> logger)
+            a = a.Replace("and", "(&|(and))", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("OVA", "((OVA)|(OAD))", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("OAD", "((OVA)|(OAD))", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("re", "re.?", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("Gekijyouban", "Gekijouban", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("to aru", "to.?aru", StringComparison.OrdinalIgnoreCase);
 
-                return false;
-            }
-            return false;
-        }
-
-        /// <summary>
-        /// Cut p(%) away from the string
-        /// </summary>
-        /// <param name="string_"></param>
-        /// <param name="min_lenght"></param>
-        /// <param name="p"></param>
-        /// <returns></returns>
-        public async static Task<string> Half_string(string string_, CancellationToken cancellationToken, int min_lenght = 0, int p = 50)
-        {
-            decimal length = 0;
-            if (await Task.Run(() => ((int)((decimal)string_.Length - (((decimal)string_.Length / 100m) * (decimal)p)) > min_lenght), cancellationToken))
-            {
-                length = (decimal)string_.Length - (((decimal)string_.Length / 100m) * (decimal)p);
-            }
-            else
-            {
-                if (string_.Length < min_lenght)
-                {
-                    length = string_.Length;
-                }
-                else
-                {
-                    length = min_lenght;
-                }
-            }
-            return string_.Substring(0, (int)length);
+            return a;
         }
 
         /// <summary>
         /// simple regex
         /// </summary>
-        /// <param name="regex"></param>
-        /// <param name="match"></param>
+        /// <param name="pattern"></param>
+        /// <param name="input"></param>
         /// <param name="group"></param>
-        /// <param name="match_int"></param>
+        /// <param name="matchInt"></param>
         /// <returns></returns>
-        public async static Task<string> One_line_regex(Regex regex, string match, CancellationToken cancellationToken, int group = 1, int match_int = 0)
+        public async static Task<string> OneLineRegex(string pattern, string input, CancellationToken cancellationToken, int group = 1, int matchInt = 0)
         {
-            Regex _regex = regex;
             int x = 0;
-            foreach (Match _match in regex.Matches(match))
+            foreach (Match match in Regex.Matches(input, pattern, RegexOptions.IgnoreCase))
             {
-                if (x == match_int)
+                if (x == matchInt)
                 {
-                    return await Task.Run(() => _match.Groups[group].Value.ToString(), cancellationToken);
+                    return await Task.Run(() => match.Groups[group].Value, cancellationToken);
```

Comment:
I know this wasn't modified from existing code... but there's no reason to have this in a `Task.Run`